### ISSUE TITLE
animate_anything_svd_v1.0

### ISSUE THREAD

**Issue by YujieOuO** [alibaba/animate-anything#17]
Good project. 

Is the animate_anything_svd_v1.0 model fine-tuned on your 20k data based on the SVD pre-trained model? 
I am curious about which modules of the model are fine-tuned.

**Comment by daizuozhuo** [comment]
For the release model, I finetuned all unet. Normally, finetune attn1, attn2, temp_conv, conv_in is enough.

**Comment by YujieOuO** [comment]
> For the release model, I finetuned all unet. Normally, finetune attn1, attn2, temp_conv, conv_in is enough.

thanks for you reply, and I want to finetune the mask model based on the SVD parameters.
Can i just use python train_svd.py --config example/train_svd_mask.yaml pretrained_model_path=**<svd_model>** ?

maybe this won't succeed because the input channel number for SVD is 8, while the current framework has 9 channels.

**Comment by daizuozhuo** [comment]
As least 20K clips, more clips will be better.

**Comment by YujieOuO** [comment]
oh, i see the convert_svd() function, 

```
unet = UNetSpatioTemporalConditionModel.from_pretrained(
        pretrained_model_path, subfolder="unet_mask", low_cpu_mem_usage=False, ignore_mismatched_sizes=True)
``` 
Perhaps this UNetSpatioTemporalConditionModel is the version after you modified the structure? Where can I find the details about the modifications?"

**Comment by daizuozhuo** [comment]
Please use this function to convert svd to svd_mask:

def convert_svd(pretrained_model_path, out_path):
    pipeline = StableVideoDiffusionPipeline.from_pretrained(pretrained_model_path)

    unet = UNetSpatioTemporalConditionModel.from_pretrained(
        pretrained_model_path, subfolder='unet', in_channels=9, low_cpu_mem_usage=False, ignore_mismatched_sizes=True)
    unet.conv_in.bias.data = copy.deepcopy(pipeline.unet.conv_in.bias)
    torch.nn.init.zeros_(unet.conv_in.weight)
    unet.conv_in.weight.data[:,1:]= copy.deepcopy(pipeline.unet.conv_in.weight)
    new_pipeline = StableVideoDiffusionPipeline.from_pretrained(
        pretrained_model_path, unet=unet)
    new_pipeline.save_pretrained(out_path)
